test-case/run-all-tests.sh

```diff
@@ -169,6 +171,10 @@ interrupted_results()
 	exit 1
 }
 
+test_kernel-boot-log()
```

Comment:
I agree with @greg-intel , please no mix in the same name.

```suggestion
test_kernel_boot_log()
```

Reply:
I'm leaning towards renaming the few tests that have underscores to be dashes, as 95% of our tests are named using dashes.  I'm not sure how the underscore tests got through, I don't remember seeing them...

Reply:
> I'm leaning towards renaming the few tests...

Consistency and simplicity are good and should absolutely be a goal for new code or code that is changing anyway for other reasons.

For unchanged code it's much more questionable. Is it worth the git churn and potential git conflicts with PRs in progress, branches, etc? It depends.